src/reporter/delete.py

```diff
@@ -33,17 +33,24 @@ def delete_entity(entity_id, type_=None, from_date=None, to_date=None):
 
 def delete_entities(entity_type, from_date=None, to_date=None,
                     drop_table=False):
-    with translator_for(fiware_s()) as trans:
-        if drop_table:
-            trans.drop_table(etype=entity_type, fiware_service=fiware_s())
-            logging.getLogger(__name__).info("dropped entity_type {}".format(entity_type))
-            return 'entity table dropped', 204
-
-        deleted = trans.delete_entities(etype=entity_type,
-                                        from_date=from_date,
-                                        to_date=to_date,
-                                        fiware_service=fiware_s(),
-                                        fiware_servicepath=fiware_sp(),)
+        with translator_for(fiware_s()) as trans:
+            if fiware_s() is not None:
```

Comment:
Shouldn't this be `fiware_sp()`?

Reply:
If you feel adventurous, perhaps it's worth renaming those functions to `fiware_service` and `fiware_service_path`, not sure what's going to break though :-)

Reply:
Hi @c0c0n3 , In already existing [code](https://github.com/orchestracities/ngsi-timeseries-api/blob/8d6d8e763d9af3dd706da8fb653c820d834ef2e5/src/reporter/delete.py#L36)  fiware_s() was used. Please confirm weather it needs to be checked for only fiware_service_path or both fiware_service or fiware_service-path? Thank you

Reply:
Hi @daminichopra, it's line 37 that needs changing I think, I should've linked the comment to that line, sorry about that! Basically I think it should be enough to change line 37 from

    if fiware_s() is not None:

to

    if fiware_sp() is not None:

note the extra `p`! Those functions have horrendous names, no wonder the confusion. I take the blame for that I think it was me who wrote those two `fiware_*` thingies...